### ISSUE TITLE
Cryteks Technique for Reducing Nausea

### ISSUE THREAD

**Issue by xEvoGx** [ElectricNightOwl/ArmSwinger#47]
Hey @kjack9 long time no speak!

I was just reading a review of **Robinson: The Journey** and they note a very interesting (and dead simple) technique for reducing movement based nausea. To simplify I'll cut and paste:

_"The player’s movement speed is controlled directly by their viewpoint. When looking directly ahead the player can move at full speed, however this momentum is then reduced when off-centre. For example, **if the player looks upwards at 45 degrees their movement speed will be halved; looking at the floor will reduce them to a crawl speed.** How does this benefit the player? Simply by reducing the chance of discomfort through unexpected travel: in first-person videogames the player can often move further than expected when not looking at the horizon, which in VR can have disastrous effects. Robinson: The Journey does not suffer from this concern-adding issue."_

This sounds like a tremendous addition to ArmSwinger imho! 

What do you think?

Cheers brother

EDIT: [Link ](https://www.vrfocus.com/2016/11/review-robinson-the-journey/)

**Comment by kjack9** [comment]
Hi there!  I will take a look at it.  At first glance, I think that having direct control of your movement speed via your hands allows you to do this naturally.  I am also hesitant to disconnect the 1:1 relationship between your hand movement and your artificial movement speed, as anything that makes you feel out of direct control seems to cause nausea issues.  But I will see what I can see.

**Comment by xEvoGx** [comment]
Fair point, and how I saw it working within your system would be a _'comfort modifier'_, explained as such that speed is reduced if looking up or down and by _n_ amount (separate values, mind). Developers could then introduce this in the _'Comfort Options'_ in their games, as there's little reason not to let a player adjust them.

Becomes another feature that we can toggle based on preference and more importantly, stomach. 😛
